src/ResourceManager/RecoveryServices.Backup/Cmdlets/Container/GetAzureRmRecoveryServicesContainer.cs

```diff
@@ -17,22 +17,48 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Management.Automation;
```

Comment:
minor - looks good if you can put them in order. also in general, group by system and followed by internal